### PR TITLE
Contact page banner and style

### PR DESCRIPTION
**Pull request by maktoob**

### CODE REVIEW

File: website/templates/website/contact.html [hunk 1]
```diff
@@ -1,17 +1,17 @@
 {% extends 'website/base.html' %}
 {% block content %}
 {% load crispy_forms_tags %}
-
-<div style="content-fluid; margin-top: 200px; margin-bottom: 160px">
-    <div class="row" style="padding-top: 100px; padding-bottom: 100px">
+<img src="/media/images/phone_.jpg" style="width:100%">
```
Comment (abunuwas) [inline website/templates/website/contact.html]: what about naming the file just `phone.jgp` in stead of `phone_.jpg`?

File: website/templates/website/contact.html [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
+<div style="content-fluid; margin-top: 50px; margin-bottom: 50px">
+    <div class="row" style="padding-left: 100px; padding-right: 100px; padding-top: 50px; padding-bottom: 50px">
```
Comment (abunuwas) [inline website/templates/website/contact.html:6]: consider moving some of these styles to a stylesheet

File: website/templates/website/contact.html (hunk 1 above)
Comment (maktoob) [reply website/templates/website/contact.html]: ok

File: website/templates/website/contact.html (hunk 2 above)
Comment (maktoob) [reply website/templates/website/contact.html:6]: I will do it later altogether.

**Review by abunuwas — APPROVED** [review]

**Pull request merged**